manifests/i/iFileSpace/iFile/2.5.9/iFileSpace.iFile.installer.yaml

```
@@ -0,0 +1,19 @@
+# Created with YamlCreate.ps1 Dumplings Mod
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: iFileSpace.iFile
+PackageVersion: 2.5.9
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: ifile.exe
+  PortableCommandAlias: ifile
+Commands:
+- ifile
+ReleaseDate: 2025-03-14
+Installers:
+- Architecture: x64
+  InstallerUrl: https://dl.ifile.space/apps/ifile_windows_amd64_2.5.9.zip
+  InstallerSha256: D4BE243A82694E9764C19453672C9751AC7F603E6EDAB1D78BE01D7472FA2E94

```

```suggestion
  InstallerSha256: 9AA2B0DE35238C592FAB45F4FE64579F0C077E7049F75AF35B9FE0398A0F1F23
```

(Automated response - build 1009.)